code/modules/jobs/job_types/wasteland.dm

```diff
@@ -239,7 +239,8 @@ Raider
 	..()
 	if(visualsOnly)
 		return
-
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/pistol127tackit)
```

Comment:
Not sure how I feel about outlaws being able to take 12.7mm pistols and give then an extra 0.15AP and a damage buff. The tactical 12.7m is a Gunrunners item rather than something any fiend could patch together.

Feels like a very unnecessary buff